modules/GKB/Release/Steps/OrthoInference.pm

```diff
@@ -5,46 +5,61 @@ use GKB::Config_Species;
 use GKB::Release::Config;
 use GKB::Release::Utils;
 
+use Carp;
 use Moose;
 extends qw/GKB::Release::Step/;
 
-has '+gkb' => ( default => "gkbdev" );
+has '+gkb' => ( default => 'gkbdev' );
 has '+passwords' => ( default => sub { ['mysql'] } );
 has '+directory' => ( default => "$release/orthoinference" );
-has '+mail' => ( default => sub { 
-                        my $self = shift;
-                        return {
-                            'to' => '',
-                            'subject' => $self->name,
-                            'body' => "",
-                            'attachment' => ""
-                        };
+has '+mail' => ( default => sub {
+    my $self = shift;
+    return {
+        'to' => '',
+        'subject' => $self->name,
+        'body' => '',
+        'attachment' => ''
+    };
 });
 has '+user_input' => (default => sub { {
-                    'skip_list_verified' => {'query' => "Has the normal event skip list been verified for version $version (y/n):"},
-                    'release_date' => {'query' => "Enter release date for version $version as yyyy-mm-dd (e.g. 2017-12-13):" }
+    'skip_list_verified' => {'query' => "Has the normal event skip list been verified for version $version (y/n):"},
+    'release_date' => {'query' => "Enter release date for version $version as yyyy-mm-dd (e.g. 2017-12-13):" },
+    'person_id' => {'query' => 'Enter person instance db id to be used for the orthoinference instance edit:'},
```

Comment:
A 'config.properties' question could also go here. If the file was populated with values from the previous release, the program would be populating the wrong DB. Or perhaps we could find a way of corroborating the current release number with the releaseNumber attribute in the code?

Reply:
Resolving, did not realize config.properties will now be built each time